src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -71,17 +85,17 @@ def kind(self) -> str:
         return "VIEW" if self.view_text is not None else "TABLE"
 
     def sql_alter_to(self, target_table_key):
-        return f"ALTER {self.kind} {self.key} SET TBLPROPERTIES ('upgraded_to' = '{target_table_key}');"
+        return f"ALTER {self.kind} {escape_sql_identifier(self.key)} SET TBLPROPERTIES ('upgraded_to' = '{target_table_key}');"
```

Comment:
'Cat'.'db'.'tbl' is different from 'cat.db.tbl'

We have to get a special key escape

Reply:
`self.key` is `catalog.database.name`, and `escape_sql_identifier` will turn escape the relevant part with backticks if needed - otherwise the `ALTER` statement will error out

our database is case insensitive, so we always lower case them